src/syslog.c

```diff
@@ -55,7 +55,11 @@ CAMLprim value caml_extunix_openlog(value v_ident, value v_option, value v_facil
   CAMLparam3(v_ident, v_option, v_facility);
   int option, facility;
   size_t index_facility;
-  char *ident;
+  static char *ident = NULL; /* openlog does _not_ store ident -- keep a heap copy */
+
+  if (NULL != ident) {
```

Comment:
I imagine the multi-threaded story of openlog is not very defined, or for that mattery, any sequence of multiple calls to openlog?

Reply:
also have to set it to NULL after free